icu4c/source/i18n/unicode/reldatefmt.h

```diff
@@ -696,11 +698,15 @@ class U_I18N_API RelativeDateTimeFormatter : public UObject {
     const SharedPluralRules *fPluralRules;
     UDateRelativeDateTimeFormatterStyle fStyle;
     UDisplayContext fContext;
+#if !UCONFIG_NO_BREAK_ITERATION
     const SharedBreakIterator *fOptBreakIterator;
+#endif // !UCONFIG_NO_BREAK_ITERATION
     Locale fLocale;
     void init(
             NumberFormat *nfToAdopt,
+#if !UCONFIG_NO_BREAK_ITERATION
             BreakIterator *brkIter,
+#endif // !UCONFIG_NO_BREAK_ITERATION
```

Comment:
```suggestion
#if UCONFIG_NO_BREAK_ITERATION
            std::nullptr_t,
#else
            BreakIterator *brkIter,
#endif // !UCONFIG_NO_BREAK_ITERATION
```
(may need `<cstddef>`)

here and in the .cpp function implementation so that call sites that already always give this a nullptr need not be made conditional.

Reply:
Done.